lib/autoload/kuroko2/workflow/notifier/concerns/chat_message_builder.rb

```diff
@@ -20,6 +20,10 @@ def launched_text
             "Launched '#{@definition.name}'"
           end
 
+          def back_to_normal_text
+            "Back to normal the current task in '#{@definition.name}'"
```

Comment:
"The task in '#{@definition.name}' was successfully retried" is easier to understand I think.